pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -691,17 +691,17 @@ func (this *SnapshotManager) CreateVolumeFromSnapshot(sourcePEID astrolabe.Prote
 		}
 		download, err = pluginClient.DatamoverV1alpha1().Downloads(veleroNs).Get(context.TODO(), downloadRecordName, metav1.GetOptions{})
 		if err != nil {
-			this.Errorf("Retrieve download record %s failed with err %v", downloadRecordName, err)
-			return false, errors.Wrapf(err, "Failed to retrieve download record %s", downloadRecordName)
+			this.Errorf("Retrieve download record %s failed with err %v, retrying..", downloadRecordName, err)
+			return false, nil
```

Comment:
How is the behavior different when returning error vs nil?  `nil` will lead to retries while error does not?